.travis.yml

```diff
@@ -8,34 +8,26 @@ before_install:
   - which diesel || cargo install diesel_cli --no-default-features --features sqlite
   - sudo cp $(which diesel) /usr/bin
 env:
-  - TEST_COMMAND="rustup component add rustfmt-preview --toolchain nightly && cargo fmt --all -- --check"
   - TEST_COMMAND="cargo build --verbose --all"
-  - TEST_COMMAND="./integration-tests/cross-build.sh"
   - TEST_COMMAND="cargo test --verbose --all" RUST_TEST_THREADS=1
-  - TEST_COMMAND="./integration-tests/rita.sh" INITIAL_POLL_INTERVAL=5 BACKOFF_FACTOR="1.5" VERBOSE=1
-  - TEST_COMMAND="./integration-tests/rita.sh" REVISION_B=release COMPAT_LAYOUT="inner_ring_old" INITIAL_POLL_INTERVAL=5 BACKOFF_FACTOR="1.5" VERBOSE=1
-  - TEST_COMMAND="./integration-tests/rita.sh" REVISION_B=master COMPAT_LAYOUT="inner_ring_old" INITIAL_POLL_INTERVAL=5 BACKOFF_FACTOR="1.5" VERBOSE=1
 rust:
   - stable
   - beta
   - nightly
 script:
   - (eval "$TEST_COMMAND")
 matrix:
-  exclude:
-  - rust: stable
-    env: TEST_COMMAND="rustup component add rustfmt-preview --toolchain nightly && cargo fmt --all -- --check"
-  - rust: beta
-    env: TEST_COMMAND="rustup component add rustfmt-preview --toolchain nightly && cargo fmt --all -- --check"
-  - rust: stable
-    env: TEST_COMMAND="./integration-tests/cross-build.sh"
-  - rust: beta
-    env: TEST_COMMAND="./integration-tests/cross-build.sh"
-  - rust: stable
-    env: TEST_COMMAND="./integration-tests/rita.sh" REVISION_B=release COMPAT_LAYOUT="inner_ring_old" INITIAL_POLL_INTERVAL=5 BACKOFF_FACTOR="1.5" VERBOSE=1
-  - rust: beta
-    env: TEST_COMMAND="./integration-tests/rita.sh" REVISION_B=release COMPAT_LAYOUT="inner_ring_old" INITIAL_POLL_INTERVAL=5 BACKOFF_FACTOR="1.5" VERBOSE=1
-  - rust: stable
-    env: TEST_COMMAND="./integration-tests/rita.sh" REVISION_B=master COMPAT_LAYOUT="inner_ring_old" INITIAL_POLL_INTERVAL=5 BACKOFF_FACTOR="1.5" VERBOSE=1
-  - rust: beta
-    env: TEST_COMMAND="./integration-tests/rita.sh" REVISION_B=master COMPAT_LAYOUT="inner_ring_old" INITIAL_POLL_INTERVAL=5 BACKOFF_FACTOR="1.5" VERBOSE=1
+  allow_failures:
+    - rust: beta
+    - rust: nightly
+  include:
+    - name: "cargo fmt"
+      script: rustup component add rustfmt-preview --toolchain nightly && cargo fmt --all -- --check
```

Comment:
this should not be toolchain nightly

Reply:
As discussed there seems to be a problem with `rustfmt` on stable. It produces a relatively big diff, which I wouldn't expect to happen. For some reason default set of settings is different on nightly, even after recent release of rust `1.28`. Lets keep it consistent at least for now. I'll just update this PR with `rustfmt.toml` that would hopefully make `rustfmt` deterministic in future. Also pre-commit hook would be nice.